service/controller/v13/version_bundle.go

```diff
@@ -17,6 +17,41 @@ func VersionBundle() versionbundle.Bundle {
 				Description: "Added CA public key into trusted user keys for SSO ssh.",
 				Kind:        versionbundle.KindAdded,
 			},
+			{
+				Component:   "aws-operator",
+				Description: "Fix volume attachments while upgrading.",
+				Kind:        versionbundle.KindFixed,
+			},
+			{
+				Component:   "aws-operator",
+				Description: "Add encrypter interface, make KMS code satisfy it and add Vault encrypter.",
+				Kind:        versionbundle.KindAdded,
+			},
+			{
+				Component:   "aws-operator",
+				Description: "Enabled migration resource with credentials logic.",
+				Kind:        versionbundle.KindAdded,
+			},
+			{
+				Component:   "aws-operator",
+				Description: "Add public for accessing Vault ELB when using Vault encrypter.",
+				Kind:        versionbundle.KindAdded,
+			},
+			{
+				Component:   "aws-operator",
+				Description: "Improve logging for shouldScale method.",
```

Comment:
Not sure we need a changelog for this. It's a low level change and not interesting to customers IMO.

Reply:
Indeed, removed.